libs/razorpay/src/services/verify-signature/service.ts

```diff
@@ -0,0 +1,60 @@
+import * as CryptoJS from 'crypto-js';
```

Comment:
Rename this service to verify-payment

Reply:
changed to process-payment as verify-payment already exist in swr